src/riak_test_escript.erl

```diff
@@ -149,6 +149,7 @@ main(Args) ->
                     Tests0;
                 {_, undefined} ->
                     Tests0;
+                %% TODO don't understand what this does
```

Comment:
I left it in there - it was something to do with parallel tests running in the cloud - but I don't understand why/how it is used...